src/domain/schemas/lesson-schemas.js

```diff
@@ -13,3 +13,8 @@ export const lessonDBSchema = joi.object({
   cdnResources: joi.array().items(joi.string()).required(),
   schedule: joi.object().allow(null)
 });
+
+export const getLessonParamsSchema = joi.object({
+  lessonId: joi.string().required(),
```

Comment:
Don't we wanna use the `idOrKeySchema` also for lessons?

Reply:
Of course, we should. Updated here and in all room schemas as well.